analytics/terraform/superset-on-eks/pv.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
move these to dedicated folder like `analytics/terraform/superset-on-eks/superset-yamls/pv.yaml`